core/client/templates/posts.hbs

```diff
@@ -17,6 +17,8 @@
                 {{#link-to "posts.post" post class="permalink" alternateActive=view.active title="Edit this post"}}
                     <h3 class="entry-title">{{post.model.title}}</h3>
                     <section class="entry-meta">
+                        <img class="avatar" src="{{unbound post.authorAvatar}}" title="{{unbound post.authorName}}" alt="">
```

Comment:
You can use `{{bind-attr}}` here:  `{{bind-attr src="post.authorAvatar" title="post.authorName"}}`.